doc/reference/devices_disk.md

```diff
@@ -92,25 +92,25 @@ Note that you cannot use initial volume configurations with custom volume option
-`path`              | string    | -         | yes       | Path inside the instance where the disk will be mounted (only for containers)
-`pool`              | string    | -         | no        | The storage pool to which the disk device belongs (only applicable for storage volumes managed by LXD)
-`propagation`       | string    | -         | no        | Controls how a bind-mount is shared between the instance and the host (can be one of `private`, the default, or `shared`, `slave`, `unbindable`,  `rshared`, `rslave`, `runbindable`,  `rprivate`; see the Linux Kernel [shared subtree](https://www.kernel.org/doc/Documentation/filesystems/sharedsubtree.txt) documentation for a full explanation) <!-- wokeignore:rule=slave -->
-`raw.mount.options` | string    | -         | no        | File system specific mount options
-`readonly`          | bool      | `false`   | no        | Controls whether to make the mount read-only
-`recursive`         | bool      | `false`   | no        | Controls whether to recursively mount the source path
-`required`          | bool      | `true`    | no        | Controls whether to fail if the source doesn't exist
-`shift`             | bool      | `false`   | no        | Sets up a shifting overlay to translate the source UID/GID to match the instance (only for containers)
-`size`              | string    | -         | no        | Disk size in bytes (various suffixes supported, see {ref}`instances-limit-units`) - only supported for the `rootfs` (`/`)
-`size.state`        | string    | -         | no        | Same as `size`, but applies to the file-system volume used for saving runtime state in VMs
-`source`            | string    | -         | yes       | Source of a file system or block device (see {ref}`devices-disk-types` for details)
+`io.cache`          | string    | `none`        | no        | Only for VMs: Override the caching mode for the device (`none`, `writeback` or `unsafe`)
+`limits.max`        | string    | -             | no        | I/O limit in byte/s or IOPS for both read and write (same as setting both `limits.read` and `limits.write`)
+`limits.read`       | string    | -             | no        | I/O limit in byte/s (various suffixes supported, see {ref}`instances-limit-units`) or in IOPS (must be suffixed with `iops`) - see also {ref}`storage-configure-IO`
+`limits.write`      | string    | -             | no        | I/O limit in byte/s (various suffixes supported, see {ref}`instances-limit-units`) or in IOPS (must be suffixed with `iops`) - see also {ref}`storage-configure-IO`
+`path`              | string    | -             | yes       | Path inside the instance where the disk will be mounted (only for containers)
+`pool`              | string    | -             | no        | The storage pool to which the disk device belongs (only applicable for storage volumes managed by Incus)
```

Comment:
There's a stray Incus in there

Reply:
Thanks for that. I also fixed a couple of commit change logs.